src/main.rs

```diff
@@ -295,113 +299,95 @@ fn main() {
         std::process::exit(exitcode::OK);
     }
 
-    #[cfg(unix)]
+    // Signalling
```

Comment:
Since, we've removed the `cfg` what is the point of this block?  This leads me to think we need to throw this into a fn or just remove this block?

Reply:
Just for clearer grouping, so yes, `fn` should be better.